lldb/source/Plugins/ABI/RISCV/ABISysV_riscv.h

```diff
@@ -124,6 +124,9 @@ class ABISysV_riscv : public lldb_private::RegInfoBasedABI {
   using lldb_private::RegInfoBasedABI::RegInfoBasedABI; // Call CreateInstance
                                                         // instead.
   bool m_is_rv64; // true if target is riscv64; false if target is riscv32
+  static constexpr size_t s_regs_for_args_count =
```

Comment:
does this need to live in the header?

Reply:
I use `s_regs_for_args_count` in 2 different `ABISysV_riscv` methods, so I've decided to place it here. Maybe I can create a static global value in `ABISysV_riscv.cpp` instead.

Reply:
yup i think that would be preferable

Reply:
addressed